decisions/log/0036-clarify-product-amount

```diff
@@ -0,0 +1,102 @@
+
+
+## Context 
+
+To share a given PCF, Framework V2 and Tech Specs require the PCF to be shared with units kgCO2e / `declaredUnit` 
+ 
+`declaredUnit` may be specified with the physical unit (liter, kilogram, cubic meter, etc.), or piece (to be released in v3) 
+ 
+To share the actual AMOUNT of declared units the product contains, the attribute `unitaryProductAmount` must be populated. 
+ 
+For example, a 12L bottle of bio-Ethanol: 
+```
+declaredUnit: liter 
+unitaryProductAmount: 12 
+pCfExcludingBiogenic: 1.5 
+```
+ 
+This data should be interpreted as the bio-ethanol has a carbon intensity of 1.5 kgCO2e / liter. The total carbon intensity of the 12L bottle (i.e. the product) has a TOTAL carbon intensity therefore of 1.5 kgCO2e x 12 = 18 kgCO2e (assuming packaging emissions are excluded). 
+ 
+
+## Problem Statement 
+ 
+We received feedback from the PACT community (specifically from suppliers requested to provide PCF data) that from a data-entry perspective, these attributes are not very clear and easily confused, which is risky given the fundamental nature of these three attributes. 
+
+For example, some suppliers entered their entire supply amount in the `unitaryProductAmount` attribute, and others entered the PCF of the ENTIRE product as `pCfExcludingBiogenic`, rather than for 1 unit of the given declared unit. 
+ 
+## Proposal 
+ 
+Introducing `“piece”` as a `declaredUnit` (see feedback item here) helps to address this problem, however it doesn’t fully address the confusion of the attributes and their interpretation especially when a physical declared unit is being used. Additionally, examples in the   PACT v2.2.0 Simplified Tech Specs Data Model_vSHARED.xlsx were created and provided to support further bringing clarity in the data exchange. 
+ 
+However, besides the above additions, the PACT Methodology working group members raised feedback to consider the naming of the attributes, which are not intuitive enough to users. 
+ 
+The specific feedback received was that part of the confusion might come from the term '*declared unit*'. ISO14067 (6.3.3 Functional or declared unit), states *“CFP study shall clearly specify the functional or declared unit of the system under study. The functional or declared unit shall be consistent with the goal and scope of the CFP study. The primary purpose of a functional or declared unit is to provide a reference to which the inputs and outputs are related. Therefore, the functional or declared unit shall be clearly defined and measurable.”*                                     
+ 
+Therefore, per ISO the '*declared unit*' is already an amount. It is not a unit of measure. PACT Tech Specs currently do not reflect this, in that declaredUnit is only a unit of measure in the specs. 
+
+### Data Model Proposal 
+ 
+Therefore, we propose the following changes to the data model: 
+ 
+- Rename `declaredUnit` to `declaredUnitOfMeasurement` 
```

Comment:
Note fix in camel-casing of declaredUnit**Of**Measurement. This is different in the working documents leading up to this proposal.

Reply:
Great catch! I just updated the word doc for consistency to declaredUnitOfMeasurement.